corehq/apps/tzmigration/tests/test_timezonemigration.py

```diff
@@ -95,11 +97,23 @@ def _compare_cases(self, actual_json, expected_json, msg):
         self.assertDictEqual(actual_json, expected_json, msg)
 
     def test_migration(self):
-        xform = self.get_xml('form')
-        form_bad_tz = self.get_json('form')
-        case_bad_tz = self.get_json('case')
-        form_good_tz = self.get_json('form-tz')
-        case_good_tz = self.get_json('case-tz')
+        def replace_ids(string_json):
+            # using format on a string json does not work well, so use replace instead
+            for replacement in (("{instance_id}", form_id), ("{case_id}", case_id)):
+                string_json = string_json.replace(*replacement)
+            return string_json
```

Comment:
Could use `%(instance_id)s` style formatting placeholders
```suggestion
            return string_json % {"instance_id": form_id, "case_id": case_id}
```